### PR TITLE
Added couchdb and mongodb packages

### PR DESCRIPTION
**Pull request by yaso195**
@earlruby @variadico @zquestz

### CODE REVIEW

File: packages/couchdb-2.0.0.conf [hunk 1]
```diff
@@ -0,0 +1,65 @@
+# Name
+#	couchdb-2.0.0.conf
+#
+# Description
+#   package for couchdb
+#
+# Installation
+#   apc package delete /apcera/pkg/packages::couchdb-2.0.0 --batch
+#   apc package build couchdb-2.0.0.conf --batch
+#
```
Comment (variadico) [inline packages/couchdb-2.0.0.conf]: Did you accidentally leave this?

File: packages/couchdb-2.0.0.conf [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+environment {
+	"PATH": "/opt/apcera/couchdb/rel/couchdb/bin:$PATH",
+	}
+
+cleanup [
+          "/root"
+        ]
+
+include_files [
+        ]
```
Comment (variadico) [inline packages/couchdb-2.0.0.conf]: Seems to be some weird spacing going on here. Also, you probably don't need `include_files`.

**Comment by variadico** [comment]
Same comments for the mongo package.

File: packages/couchdb-2.0.0.conf (hunk 1 above)
Comment (yaso195) [inline packages/couchdb-2.0.0.conf]: no those are just the comments to show how to delete and build this package. We better have them to show how to build. You can find the same comments in mongodb package.

**Comment by yaso195** [comment]
Done, made the same changes for mongodb

**Comment by zquestz** [comment]
LGTM. Also for mongo, would have loved to use https://www.percona.com/software/mongo-database/percona-server-for-mongodb as it is WAY better than stock mongodb. For another day. =)

**Pull request merged**